lib/urcli-money.js

```diff
@@ -132,15 +139,22 @@ class Report {
     console.log(chalk.blue(`Earnings Report for ${startDate} to ${endDate}:`))
 
     for (let p in this.projects) {
-      let {name, id, ungradeable, passed, failed, earned} = this.projects[p]
+      let {name, id, ungradeable, passed, failed, earned, hours} = 
+          this.projects[p]
       console.log(`
       ${chalk.white(`Project: ${name} (${id}):`)}
           ${chalk.white(`Total reviewed: ${passed + failed}`)}
           ${chalk.white(`Ungradeable: ${ungradeable}`)}
           ${chalk.white(`Earned: ${earned}`)}
+          ${chalk.white(`Working Hours: ${hours.toFixed(2)}`)}
```

Comment:
I don't think one can assume that this is the actual time that has been put in. If someone held on to a review for 7 hours and the review only took 10 mins, it would skew the results badly. Perhaps we could label this differently?